.github/workflows/ci.yml

```diff
@@ -5,47 +5,42 @@ on:
     branches:
       - master
   pull_request:
-    branches:
-      - "*"
 
 jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - uses: dtolnay/rust-toolchain@nightly
+      - run: cargo test
 
   format:
     name: Format
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@nightly
         with:
-          toolchain: stable
           components: rustfmt
-      - uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: -- --check
+      - run: cargo fmt -- --check
 
   clippy:
     name: Clippy
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@nightly
```

Comment:
Same as above.

Reply:
this one should be nightly. Clippy nightly has access to significantly more lints. The `rust-version` config option in `Cargo.toml` prevents clippy from making suggestions including syntax not supported by the MSRV, so there's no reason not to use a more recent version of clippy